Cli-CredentialHelper/install.cmd

```diff
@@ -56,5 +56,3 @@ IF "%~1" EQU "/?" (
     "%~dp0\git-credential-manager" ?
 
     EXIT /B 0
-
```

Comment:
The author kept two extra line-endings after each sub-routine. I do believe these lines are rather intentional and we should likely not be removing them.